### PR TITLE
made gamemode kill cava and waybar, set volume max to 150%

### PR DESCRIPTION
**Pull request by Sipper1236**
Pull Request
Description

Please read these instructions and remove unnecessary text.

    Try to include a summary of the changes and which issue is fixed.
    Also include relevant motivation and context (if applicable).
    List any dependencies that are required for this change. (e.g., packages or other PRs)
    Provide a link if there is an issue related to this pull request. e.g., Fixes # (issue)
    Please add Reviewers, Assignees, Labels, Projects, and Milestones to the PR. (if applicable)

Type of change

Please put an x in the boxes that apply:

Bug fix (non-breaking change which fixes an issue)
[X ] New feature (non-breaking change which adds functionality)
Breaking change (fix or feature that would cause existing functionality to not work as expected)
Documentation update (non-breaking change; modified files are limited to the documentations)

    Technical debt (a code change that does not fix a bug or add a feature but makes something clearer for devs)
    [ X] Other (provide details below)
    Improves performance

Checklist

Please put an x in the boxes that apply:

    [ X] I have read the [CONTRIBUTING](https://github.com/JaKooLit/Hyprland-Dots/blob/main/CONTRIBUTING.md) document.

My code follows the code style of this project.
[ X] My commit message follows the [commit guidelines](https://github.com/JaKooLit/Hyprland-Dots/blob/main/CONTRIBUTING.md#git-commit-messages).
[ X] My change requires a change to the documentation.
I want to add something in Hyprland-Dots wiki.

    I have added tests to cover my changes.
    [ X] I have tested my code locally and it works as expected.
    [ X] All new and existing tests passed.

Screenshots

(if appropriate)
Additional context
there may be  a need for a new picture for when the volume is above 100%
image

### CODE REVIEW

File: config/hypr/scripts/Volume.sh [hunk 1]
```diff
@@ -44,7 +44,7 @@ inc_volume() {
     if [ "$(pamixer --get-mute)" == "true" ]; then
         pamixer -u && notify_user
     fi
-    pamixer -i 5 && notify_user
+    pamixer -i 5 --allow-boost --set-limit 150 && notify_user
```
Comment (JohnRTitor) [inline config/hypr/scripts/Volume.sh:47]: Note: Pavucontrol already allows volume boost upto 150%, so adding this won't hurt.

File: config/hypr/scripts/GameMode.sh [hunk 2]
```diff
@@ -16,15 +16,17 @@ if [ "$HYPRGAMEMODE" = 1 ] ; then
         keyword general:gaps_out 0;\
         keyword general:border_size 1;\
         keyword decoration:rounding 0"
-    swww kill 
+    swww kill
+    
+done
```
Comment (JohnRTitor) [inline config/hypr/scripts/GameMode.sh:21]: Why `done` in a `if` statement?

**Comment by JaKooLit** [comment]
dang hard to fix the conflict lmao... im closing this and upload the Volume.sh which I am using.. No worries I will credit you

File: config/hypr/scripts/Volume.sh (hunk 1 above)
Comment (JohnRTitor) [reply config/hypr/scripts/Volume.sh:47]: I know :) saw that.

**Pull request closed without merging**